scripts/security/certs-create-per-user.sh

```diff
@@ -19,6 +19,12 @@ keytool -genkey -noprompt \
 keytool -keystore kafka.$i.keystore.jks -alias $i -certreq -file $i.csr -storepass confluent -keypass confluent -ext "SAN=dns:$i,dns:localhost"
 #openssl req -in $i.csr -text -noout
 
+# Enables 'confluent login --ca-cert-path /etc/kafka/secrets/snakeoil-ca-1.crt --url https://kafka1:8091'
+DNS_ALT_NAMES=$(printf '%s\n' "DNS.1 = $i" "DNS.2 = localhost")
+if [[ "$i" == "mds" ]]; then
+  DNS_ALT_NAMES=$(printf '%s\n' "$DNS_ALT_NAMES" "DNS.3 = kafka1" "DNS.4 = kafka2")
+fi
```

Comment:
I don't have context for the full PR, but this looks like an elegant way to do this.